website/src/app/[lang]/[region]/(website)/campaign/[campaign]/page.tsx

```diff
@@ -47,127 +65,177 @@ export default async function Page({ params }: CampaignPageProps) {
 
 	return (
 		<>
-			<BaseContainer className="-mx-2 flex flex-wrap">
-				<div className="mb-4 w-full px-2 lg:w-1/2">
-					<Typography size="xl" weight="bold" color="accent">
-						{translator.t('campaign.by', { context: { creator: campaign.creator_name } })}
-					</Typography>
-					<Typography size="5xl" weight="bold" color="accent" className="mt-2">
-						{campaign.title}
-					</Typography>
-				</div>
-			</BaseContainer>
-			<BaseContainer className="-mx-2 flex flex-wrap">
-				<div className="mb-4 w-full px-2 lg:w-1/2">
-					<Typography size="lg" weight="medium">
-						{campaign.description}
-					</Typography>
-				</div>
-				<div className="mb-4 w-full px-2  lg:w-1/2">
-					<div>
-						{!campaign.goal && (
-							<div className="mb-4 flex flex-col items-center">
-								<Typography size="2xl" weight="bold" color="accent">
-									{translator?.t('campaign.without-goal.collected', {
-										context: {
-											count: contributions,
-											amount: amountCollected,
-											currency: campaign.goal_currency,
-											total: campaign.goal,
-										},
-									})}
+			<BaseContainer>
+				<div className="py-12">
+					<div className="grid grid-cols-1 gap-20 lg:grid-cols-2">
+						<div className="flex flex-col justify-center gap-3">
+							<div>
+								<Typography size="xl" color="primary">
+									{translator.t('campaign.by', { context: { creator: campaign.creator_name } })}
+								</Typography>
+							</div>
+							<div>
+								<Typography weight="medium" color="primary" style={{ lineHeight: '70px' }} className="mt-2 text-[4rem]">
+									{campaign.title}
+								</Typography>
+							</div>
+							<div className="my-8">
+								<Typography size="2xl" color="primary">
+									{campaign.description}
 								</Typography>
 							</div>
-						)}
-						{percentageCollected !== undefined && (
-							<div className={'mb-6'}>
-								<div className=" mb-1 flex font-medium">
-									<div className="text-accent w-1/2">
+							<div className="flex">
```

Comment:
Same as above, wrapping everything which is progress bar related in a `{percentageCollected !== undefined && (` would be good to support non goal campaigns:

<img width="573" alt="image" src="https://github.com/socialincome-san/public/assets/1204548/8add885b-6efa-421e-9a85-669c285204f6">